atd-vzd/config.yaml

```diff
@@ -1,6 +1,5 @@
 version: 3
-endpoint: http://localhost:8084
-admin_secret: thisisasecret
```

Comment:
Thanks for removing this! We really need to remove this secret that I added to the docker compose file a while back. It prevents VZE from connecting to the Hasura instance in the non-LDM branches.

Reply:
Oh cool, that was all Hasura's work during the `init` command process.